apluslms_roman/builder.py

```diff
@@ -60,11 +64,11 @@ def __init__(self, backend_class=None, settings=None):
 
         name = getattr(backend_class, 'name', None) or backend_class.__name__.lower()
         env_prefix = name.upper() + '_'
-        env = {k: v for k, v in environ.items() if k.startswith(env_prefix)}
+        env = {key: val for key, val in environ.items() if key.startswith(env_prefix)}
```

Comment:
Even though this would be a good change, can you leave it as it's now. #8 is rewriting this part any case.